src/sentry/integrations/slack/integration.py

```diff
@@ -189,6 +190,32 @@ def build_integration(self, state):
 
         return integration
 
+    def create_audit_log_entry(self, integration, organization, request, action, extra=None):
+        if action == "upgrade":
+            create_audit_entry(
+                request=request,
+                organization=organization,
+                target_object=integration.id,
+                event=AuditLogEntryEvent.INTEGRATION_UPGRADE,
+                data={"provider": integration.provider, "name": integration.name},
+            )
+
+            if extra and extra.get("extra_orgs"):
+                for org in Organization.objects.filter(slug__in=extra["extra_orgs"]):
+                    create_audit_entry(
+                        request=request,
+                        organization=org,
+                        target_object=integration.id,
+                        event=AuditLogEntryEvent.INTEGRATION_UPGRADE,
+                        data={"provider": integration.provider, "name": integration.name},
+                    )
+
+            return
+
+        super(SlackIntegrationProvider, self).create_audit_log_entry(
```

Comment:
@MeredithAnya Nit: I think it's more readable to call super at the top so it's more clear to people upon initial inspection that it is using super.